insights/specs/datasources/ssl_certificate.py

```diff
@@ -54,17 +77,30 @@ def httpd_ssl_certificate_files(broker):
     Raises:
         SkipComponent: Raised if "SSLCertificateFile" directive isn't found
     """
-    conf = broker[HttpdConfTree]
-    virtual_hosts = conf.find('VirtualHost')
-    ssl_certs = []
-    for host in virtual_hosts:
-        ssl_cert = ssl_engine = None
-        ssl_engine = host.select('SSLEngine')
-        ssl_cert = host.select('SSLCertificateFile')
-        if ssl_engine and ssl_engine.value and ssl_cert:
-            ssl_certs.append(str(ssl_cert.value))
+    confs = broker[LocalSpecs.httpd_files]
+    ssl_engine = ssl_cert = None
+    ssl_certs = set()
+    virtual_host_start = False
+    for conf in confs:
+        if os.path.exists(conf):
```

Comment:
The existence of the `conf` file was already guaranteed by the `httpd.httpd_configuration_files`, this check is not required too, and the code can be simplified a bit after removing it.